tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -640,76 +644,100 @@ describe(__filename, () => {
     });
 
-    it('handles a submitted review', () => {
-      const addonSlug = 'some-slug';
-      dispatchAddon({ ...fakeAddon, slug: addonSlug });
-      const dispatchSpy = sinon.spy(store, 'dispatch');
-      const root = render({ reviews: null, params: { addonSlug } });
+    it('renders RatingsByStar without an add-on', () => {
+      const root = render({ addon: null });
+      const ratingsByStar = root.find(RatingsByStar);
 
-      const manager = root.find(RatingManager);
-      expect(manager).toHaveProp('onReviewSubmitted');
-      const onReviewSubmitted = manager.prop('onReviewSubmitted');
+      expect(ratingsByStar).toHaveProp('addon', null);
+    });
 
-      // Simulate a review submission.
-      onReviewSubmitted();
+    it('renders RatingsByStar with an add-on', () => {
+      const addon = { ...fakeAddon };
+      dispatchAddon(addon);
+      const root = render();
+      const ratingsByStar = root.find(RatingsByStar);
 
-      sinon.assert.calledWith(dispatchSpy, clearAddonReviews({ addonSlug }));
+      expect(ratingsByStar).toHaveProp('addon');
+      expect(ratingsByStar.prop('addon')).toMatchObject({ id: addon.id });
     });
 
-    it('resets the page after submitting a review', () => {
-      const history = createFakeHistory();
-      const props = {
-        history,
-        location: createFakeLocation({
-          query: { page: 2 },
-        }),
-        params: {
-          addonSlug: fakeAddon.slug,
-        },
-        reviews: null,
-      };
+    it('renders loading text without an add-on', () => {
+      const root = render({ addon: null });
 
-      dispatchAddon({ ...fakeAddon });
+      expect(
+        root.find('.AddonReviewList-addonAverage').find(LoadingText),
+      ).toHaveLength(1);
+    });
+
+    it('renders empty text without add-on ratings', () => {
+      dispatchAddon({ ...fakeAddon, ratings: undefined });
+      const root = render();
 
-      const root = render(props);
+      expect(root.find('.AddonReviewList-addonAverage').text()).toEqual('');
+    });
 
-      const manager = root.find(RatingManager);
-      expect(manager).toHaveProp('onReviewSubmitted');
-      const onReviewSubmitted = manager.prop('onReviewSubmitted');
+    it('renders a fixed star average', () => {
+      dispatchAddon({
+        ...fakeAddon,
+        ratings: {
+          ...fakeAddon.ratings,
+          average: 4.6667,
+        },
+      });
+      const root = render();
 
-      // Simulate a review submission.
-      onReviewSubmitted();
+      expect(root.find('.AddonReviewList-addonAverage').text()).toEqual(
+        '4.7 star average',
+      );
+    });
 
-      sinon.assert.calledWith(history.push, {
-        pathname: `/${lang}/${clientApp}${root.instance().url()}`,
-        query: { page: 1 },
+    it('renders whole number star averages', () => {
+      dispatchAddon({
+        ...fakeAddon,
+        ratings: {
+          ...fakeAddon.ratings,
+          average: 4.0,
```

Comment:
Nit: Would it be better to use a `const` for this and in the assertion?

Reply:
Same reason as above.